src/converter/jfr2pprof.java

```diff
@@ -56,8 +60,20 @@ public boolean equals(final Object other) {
         }
     }
 
+    public static final String TYPE_CPU = "cpu";
+    public static final String TYPE_ALLOC = "alloc";
+    public static final String TYPE_LOCK = "lock";
+
     public static final byte[] METHOD_UNKNOWN = "[unknown]".getBytes();
 
+    public static final byte[] VALUE_TYPE_CPU = "cpu".getBytes(StandardCharsets.UTF_8);
+    public static final byte[] VALUE_TYPE_ALLOC = "allocations".getBytes(StandardCharsets.UTF_8);
+    public static final byte[] VALUE_TYPE_LOCK = "lock".getBytes(StandardCharsets.UTF_8);
+
+    public static final byte[] VALUE_UNIT_CPU = "nanoseconds".getBytes(StandardCharsets.UTF_8);
+    public static final byte[] VALUE_UNIT_ALLOC = "count".getBytes(StandardCharsets.UTF_8);
+    public static final byte[] VALUE_UNIT_LOCK = "nanoseconds".getBytes(StandardCharsets.UTF_8);
```

Comment:
How come the value is "count" for allocations, but "nanoseconds" for locks? I see the converter code handles both events exactly the same way.
Also, does it make sense to include "bytes" in the allocation profile?